lib/rest/client.js

```diff
@@ -462,6 +464,25 @@ class Client {
         // Call callback with a single page of data. If callback returns true,
         // cancel the listing.
         const children = json.children || [];
+
+        // Convert timestamps: 2020-04-16T09:57:00 -> milliseconds UTC
+        if (json) {
+          if (json.time) {
+            // eslint-disable-next-line no-param-reassign
+            json.time = moment.parseZone(json.time + TZ)
+              .utc()
+              .valueOf();
+          }
+
+          for (let i = 0; i < children.length; i++) {
+            // eslint-disable-next-line no-continue
+            if (!children[i].time) continue;
+            children[i].time = moment.parseZone(children[i].time + TZ)
```

Comment:
This seems to be identical to above. You might want to make it a function

Reply:
Good call, inbound.